Gemfile

```diff
@@ -69,6 +69,7 @@ group :test do
   gem 'timecop'
   gem 'capybara-screenshot'
   gem 'minitest-ci'
+  gem 'rails-controller-testing'
```

Comment:
Yeah, we might be able to. This was mainly a shim to allow us to use the old deprecated style.